cluster/gce/config-default.sh

```diff
@@ -53,6 +53,7 @@ ENABLE_DOCKER_REGISTRY_CACHE=true
 # Optional: Enable node logging.
 ENABLE_NODE_LOGGING=true
 LOGGING_DESTINATION=elasticsearch # options: elasticsearch, gcp
+ELASTICSEARCH_LOGGING_REPLICAS=1
```

Comment:
You should guard this with ENABLE_CLUSTER_LOGGING, similar to how monitoring is split.  I know it's a plumbing mess...